internal/services/web/static_site_resource_test.go

```diff
@@ -77,13 +77,14 @@ func TestAccAzureStaticSite_identity(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
-		{
-			Config: r.basic(data),
-			Check: acceptance.ComposeTestCheckFunc(
-				check.That(data.ResourceName).ExistsInAzure(r),
-			),
-		},
-		data.ImportStep(),
+		// There is API issue to downgrade the sku to Free with identity configured: https://github.com/Azure/azure-rest-api-specs/issues/18253
```

Comment:
can we make this a TODO:

```suggestion
		// TODO: re-enable this once the API issue is resolved: https://github.com/Azure/azure-rest-api-specs/issues/18253
		// API currently doesn't allow downgrading to a Free SKU with an Identity attached
```